types/castv2/index.d.ts

```diff
@@ -0,0 +1,119 @@
+/// <reference types="node" />
+
+import { EventEmitter } from "events";
```

Comment:
```suggestion
import { EventEmitter } from "node:events";
```
if you want to be explicit, which 'events' are to be used (regarding dependency entry)

Reply:
Makes sense, thanks!